backend/app/routes/Q&A/answers/index.js

```diff
@@ -12,7 +12,7 @@ const updateAnswerStatus = require('./updateAnswerStatus');
 router.post('/', validation(answerValidationSchema), postAnswer);
 
 // GET API FOR ANSWERS
-router.get('/', validation(getAnswerValidationSchema), getAnswers);
+router.post('/getAnswer', validation(getAnswerValidationSchema), getAnswers);
```

Comment:
[question] why did you convert it to `post`? its an api to fetch data from db, so it should be a get route

Reply:
But it is taking the data. instead of changing entire code I changed the little bit. So it works fine.

Reply:
then we will fix the api, changing get to post is wrong. The change will be very small, check https://github.com/HITK-TECH-Community/Community-Website/pull/884/files for reference